examples/basic-http/src/main/java/io/cloudeventrs/examples/http/HttpServer.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2018-Present The CloudEvents Authors
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.cloudeventrs.examples.http;
+
+import com.sun.net.httpserver.Headers;
+import com.sun.net.httpserver.HttpExchange;
+import io.cloudevents.CloudEvent;
+import io.cloudevents.core.message.MessageReader;
+import io.cloudevents.core.message.MessageWriter;
+import io.cloudevents.http.HttpMessageFactory;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.net.InetSocketAddress;
+
+public class HttpServer {
+
+    public static void main(String[] args) throws IOException {
+        com.sun.net.httpserver.HttpServer httpServer = com.sun.net.httpserver.HttpServer.create(new InetSocketAddress("localhost", 8080), 0);
+        httpServer.createContext("/echo", HttpServer::handle);
+        httpServer.start();
+    }
+
+    public static void handle(HttpExchange exchange) throws IOException {
```

Comment:
maybe `echoHandler`?

Reply:
Can you add a bunch of comments (nothing particularly verbose), in particular when you use the sdk methods?